awx/main/management/commands/register_default_execution_environments.py

```diff
@@ -84,21 +84,28 @@ def handle(self, *args, **options):
 
             registry_cred.inputs = inputs
             registry_cred.save()
-            changed = True
-
-            if not created:
-                print('Updated Default Execution Environment Credential')
-
-        for ee in reversed(settings.DEFAULT_EXECUTION_ENVIRONMENTS):
-            _, created = ExecutionEnvironment.objects.update_or_create(
-                name=ee["name"], defaults={"image": ee["image"], "managed_by_tower": True, "credential": registry_cred}
-            )
-
-        if created:
-            changed = True
-            print("Default Execution Environment(s) registered.")
+            changed.append(True)
+
+            if not cred_created:
+                print("Updated 'Default Execution Environment Credential'")
+
+        # Create default globally available Execution Environments
+        for ee in reversed(settings.GLOBAL_JOB_EXECUTION_ENVIRONMENTS):
+            _, ee_created = ExecutionEnvironment.objects.update_or_create(name=ee["name"], image=ee["image"], credential=registry_cred)
+            if ee_created:
+                changed.append(True)
+                print(f"'{ee['name']}' Default Execution Environment registered.")
+
+        # Create the control plane execution environment that is used for project updates and system jobs
```

Comment:
not used for system jobs, right?

but is used for inventory updates, for all the default cloud sources like ec2, azure, etc.

Reply:
yeah I'll fix that comment....
but FWIW  currently the project update resolve EE method is the ONLY one that keys on managed_by_tower=True

Reply:
You're right in that project update goes straight to the global jobs default

https://github.com/ansible/awx/blob/2ce276455ad8bf252607759f8070b9f5f8f10838/awx/main/models/projects.py#L192

but all the other models fallback to the same thing

https://github.com/ansible/awx/blob/3e5ee9d57a2bad3f78141e5a17842fd9483141f5/awx/main/models/mixins.py#L489

if there's not an EE specifically set (which I would consider to be the default mode of operation)

Reply:
Scratch my last comment, project/job/inventory default is in flux with #10378

Reply:
Based on my understanding of what @shanemcd said, Project Updates will continue to be the only job type that gets set to use the control plane EE.